FlowCrypt/Controllers/Msg/MessageViewController.swift

```diff
@@ -14,7 +14,7 @@ final class MessageViewController: TableNodeViewController {
     }
 
     enum Parts: Int, CaseIterable {
-        case sender, subject, text
+        case sender, subject, text, attachment
```

Comment:
are all attachments goes under the text by design? 

can it be the case that text will go after an attachment?
text
attachment
text

?

Reply:
Right now, we'll not worry about messages that have attachments in other positions - we'll simply render them always at the end. Later (~ year 2023) we can revisit that.